tsconfig.base.json

```diff
@@ -66,7 +66,6 @@
       "@o3r/stylelint-plugin": ["packages/@o3r/stylelint-plugin/src"],
       "@o3r/styling": ["packages/@o3r/styling/src/public_api"],
       "@o3r/testing": ["packages/@o3r/testing/src/public_api"],
-      "@o3r/testing/*": ["packages/@o3r/testing/src/*"],
```

Comment:
Not sure you should remove it.
Without this rules, the deep imports will be rerouted to src/ folder in this monorepo